File: OC_project/OC_project/ViewController.m
//
//  ViewController.m
//  OC_project
//
//  Created by Lugick Wang on 2021/8/24.
//

#import "ViewController.h"
#import <test_framework/test_framework.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    OC * oc = [OC new];
    [oc oc_print];
    
    Swift *swift = [Swift new];
    [swift swift_print];
    
    
    // Do any additional setup after loading the view.
}


@end

File: Swift_project/Swift_project/ViewController.swift
//
//  ViewController.swift
//  Swift_project
//
//  Created by Lugick Wang on 2021/8/24.
//

import UIKit
import test_framework

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let oc = OC()
        oc.oc_print()
        
        let s = Swift()
        s.swift_print()
        // Do any additional setup after loading the view.
    }


}

